<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="URDF_LINKS">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="0.033059 0.00013973 0.2467"
        rpy="0 0 0" />
      <mass
        value="1.2926" />
      <inertia
        ixx="0.0026462"
        ixy="5.3985E-07"
        ixz="2.6006E-05"
        iyy="0.0017881"
        iyz="1.1365E-07"
        izz="0.0015096" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="link_1">
    <inertial>
      <origin
        xyz="-0.0083282 -0.03784 -0.024143"
        rpy="0 0 0" />
      <mass
        value="0.022157" />
      <inertia
        ixx="4.2503E-06"
        ixy="-4.1295E-07"
        ixz="-2.0719E-07"
        iyy="6.7055E-06"
        iyz="-8.6884E-07"
        izz="6.6972E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint1"
    type="revolute">
    <origin
      xyz="0.00995 -0.15355 0.26905"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="link_1" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-1.57"
      upper="1.57"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link_2">
    <inertial>
      <origin
        xyz="-0.0048734 0.020002 -0.04324"
        rpy="0 0 0" />
      <mass
        value="0.060493" />
      <inertia
        ixx="4.1778E-05"
        ixy="2.5101E-08"
        ixz="-4.0235E-06"
        iyy="4.4435E-05"
        iyz="1.858E-08"
        izz="2.9556E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint2"
    type="revolute">
    <origin
      xyz="0 -0.01998 -0.03175"
      rpy="0 0 0" />
    <parent
      link="link_1" />
    <child
      link="link_2" />
    <axis
      xyz="0 -1 0" />
    <limit
      lower="-1.57"
      upper="1.57"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link_3">
    <inertial>
      <origin
        xyz="0.00076657 -0.029227 -0.08874"
        rpy="0 0 0" />
      <mass
        value="0.10841" />
      <inertia
        ixx="2.6914E-05"
        ixy="9.7686E-08"
        ixz="1.3013E-06"
        iyy="2.9069E-05"
        iyz="-5.9723E-07"
        izz="1.4156E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint3"
    type="revolute">
    <origin
      xyz="-0.00786 0.045 -0.10972"
      rpy="0 0 0" />
    <parent
      link="link_2" />
    <child
      link="link_3" />
    <axis
      xyz="0 -1 0" />
    <limit
      lower="-1.57"
      upper="1.57"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link_5">
    <inertial>
      <origin
        xyz="-0.0083282 0.038457 -0.023332"
        rpy="0 0 0" />
      <mass
        value="0.022157" />
      <inertia
        ixx="4.3922E-06"
        ixy="4.1295E-07"
        ixz="-2.1535E-07"
        iyy="6.8473E-06"
        iyz="9.0995E-07"
        izz="6.6972E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_5.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_5.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint5"
    type="revolute">
    <origin
      xyz="0.00995 0.15355 0.26905"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="link_5" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-1.57"
      upper="1.57"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link_6">
    <inertial>
      <origin
        xyz="-0.0031295 -0.019998 -0.043401"
        rpy="0 0 0" />
      <mass
        value="0.060493" />
      <inertia
        ixx="4.2081E-05"
        ixy="2.433E-08"
        ixz="-3.5191E-06"
        iyy="4.4435E-05"
        iyz="1.9582E-08"
        izz="2.9252E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_6.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_6.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint6"
    type="revolute">
    <origin
      xyz="0 0.01998 -0.03025"
      rpy="0 0 0" />
    <parent
      link="link_5" />
    <child
      link="link_6" />
    <axis
      xyz="0 -1 0" />
    <limit
      lower="-1.57"
      upper="1.57"
      effort="200"
      velocity="3" />
  </joint>
  <link
    name="link_7">
    <inertial>
      <origin
        xyz="0.00076657 -0.029227 -0.08874"
        rpy="0 0 0" />
      <mass
        value="0.10841" />
      <inertia
        ixx="2.6914E-05"
        ixy="9.7686E-08"
        ixz="1.3013E-06"
        iyy="2.9069E-05"
        iyz="-5.9723E-07"
        izz="1.4156E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_7.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mi_br/meshes/link_7.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint7"
    type="revolute">
    <origin
      xyz="-0.0034387 0.005 -0.10995"
      rpy="0 0 0" />
    <parent
      link="link_6" />
    <child
      link="link_7" />
    <axis
      xyz="0 -1 0" />
    <limit
      lower="-1.57"
      upper="1.57"
      effort="200"
      velocity="3" />
  </joint>
</robot>
